openedx/core/djangoapps/catalog/utils.py

```diff
@@ -287,6 +287,67 @@ def get_course_runs_for_course(course_uuid):
         return []
 
 
+def get_course_uuid_for_course(course_run_key):
```

Comment:
Seems like this should be somewhere on whatever we have for a course model in the platform.

Reply:
I am not following your question here?  We made the decision early on to not include the Course key->Course UUID mapping in platform as it already exists in the Discovery service.  It seemed like a bad idea to build another copy that could become inconsistent.

Reply:
Currently LMS does not have an understanding of what a Course is.  Looking at this it is possible that we add the UUID to the Course Overview table and add it to what fills the Course Overview table.  We should talk about this on Tuesday.

Reply:
I'd be happy to talk about this Tuesday, I'm just hesitant to use an api call exclusively for a uuid lookup. If thats definitely our architecture I'm fine with it but I'd like an in person chat :)

Reply:
I was hesitant as well.  I also think that it might be of value in the LMS model depending on how often this conversion is needed. We might want to do discovery and add a story.

Reply:
Sounds good, lets talk Tuesday. I think i'm hesitant to have an api call for a simple uuid lookup, but I understand that may be one of the only ways.

Reply:
Resolved with in person conversation, @staubina  we mention documenting the tradeoffs around this call/adding course_uuid to the courseOverview (i think) model.